bool value      # true to enable, false to disable
---
